src/eclingo/parsing/transformers/ast_reify.py

```diff
@@ -0,0 +1,11 @@
+import clingo
+from clingo import ast
+from clingo.ast import (
+    AST,
+    ASTType,
+)
+
+def symbolic_literal_to_term(x: AST) -> AST:
+    assert x.ast_type == ASTType.Literal and x.atom.ast_type == ASTType.SymbolicAtom
+    arg = ast.SymbolicTerm(x.location, clingo.Function('b', [], True))
```

Comment:
Is there a reason for using 'b' and "a" when creating the SymbolicTerm and Function as placeholders?

Reply:
So, before I start looking into how to reify a Literal into a Function, I just wanted to know that this code above is arbitry right?

Reply:
'a' and 'b' are just the names.

This is specific code to pass exactly one test and check that the test make sense.

It does not transform ```x``` into anything. It just creates a new term that on purpose coincide with the first test. Obviously, any other test will fail.